lib/email-validation/strategy.rb

```diff
@@ -17,11 +17,10 @@ def verify_email(email)
         end
       end
 
-      validation_result ||= ValidationResult.new(true, true)
-
-      BlacklistedEmail.create(email: email, origin: validation_result.reason) unless validation_result.valid?
-
-      msg = validation_result.valid? ? '' : EmailValidation::incorrect_email_message(email)
+      unless validation_result.valid?
```

Comment:
That was my first though too 'case personally unless is confusing to me but one of the ruby style guide I found suggests: https://github.com/bbatsov/ruby-style-guide#no-bang-bang